ledger-service/http-json/src/main/scala/com/digitalasset/http/util/ErrorOps.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.http.util
+
+import com.digitalasset.util.ExceptionOps
+import scalaz.syntax.show._
+import scalaz.{Show, \/}
+
+object ErrorOps {
+
+  implicit final class `\\/ WSS extras throwable`[R](private val self: Throwable \/ R)
+      extends AnyVal {
+    def liftErr[M](f: String => M): M \/ R =
+      self leftMap (e => f(ExceptionOps.getDescription(e)))
+  }
+
+  implicit final class `\\/ WSS extras`[L, R](private val self: L \/ R) extends AnyVal {
+    def liftErr[M](f: String => M)(implicit L: Show[L]): M \/ R =
```

Comment:
You can curry this if you want, just know that it provides no inference benefit to do so.